ipn/ipnlocal/local_test.go

```diff
@@ -171,6 +184,17 @@ func TestShrinkDefaultRoute(t *testing.T) {
 				t.Errorf("shrink(%q).Contains(%v) = true, want false", test.route, ip)
 			}
 		}
+		ips, _, err := interfaces.LocalAddresses()
+		if err != nil {
+			t.Fatal(err)
+		}
+		for _, ip := range ips {
+			want := test.localIPFn(ip)
+			t.Logf("shrink(%q).Contains(%v) ? %v", test.route, ip, want)
```

Comment:
I haven't ever seen this ? notation. delete this? debug line?

Reply:
Oops, was debug noise to prove to myself that the check was checking for the right thing. Removed.